backend/app.py

```diff
@@ -27,11 +27,13 @@
     db_url = db_url.replace("postgres://", "postgresql://", 1)
 app.config["SQLALCHEMY_DATABASE_URI"] = db_url
 app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.config["SECRET_KEY"] = os.getenv("SECRET_KEY", "SET_A_SECRET_KEY")
 
 
 @app.route("/login", methods=["POST"])
 def login():
-
+    session.pop("favorites", default=None)
+    session.pop("state", default=None)
```

Comment:
Is this referring to sessionStorage? If so this probably wouldn't need to be invoked here since session variables clear themselves when app is closed. This would probably go where logout functionality is.

Reply:
It was just making sure that if the user started to login again, that these values will either be filled with the new login information, or cleared if there was an issue logging in